inference-engine/src/vpu/graph_transformer/src/frontend/ShaveElfMetadataParser.cpp

```diff
@@ -65,6 +65,7 @@ uint32_t md_parser_t::evaluate_expr(const md_expr_t *expression,
     case md_type_op_xor:
     case md_type_op_shl:
     case md_type_op_lshr:
+      assert(values.size() >= 2);
```

Comment:
Don't use simple `assert` - it isn't evaluated outside of `Debug` mode - use dedicated macros like `VPU_THROW_UNLESS` with descriptive messages